src/BitPaySDK/Client.php

```diff
@@ -1007,379 +987,446 @@ public function deletePayoutRecipient(string $recipientId): bool
-            throw new PayoutQueryException(
-                "failed to deserialize BitPay server response (Payout) : ".$e->getMessage());
+            throw new PayoutBatchQueryException(
+                "failed to deserialize BitPay server response (PayoutBatch) : ".$e->getMessage());
         }
 
-        return $payout;
+        return $batches;
     }
 
     /**
-     * Cancel a BitPay Payout.
+     * Cancel a BitPay Payout batch.
      *
-     * @param $payoutId string The id of the payout to cancel.
-     * @return Payout A BitPay generated Payout object.
-     * @throws PayoutCancellationException BitPayException class
+     * @param $batchId string The id of the batch to cancel.
+     * @return PayoutBatch A BitPay generated PayoutBatch object.
+     * @throws PayoutBatchCancellationException PayoutBatchCancellationException class
      */
-    public function cancelPayout(string $payoutId): bool
+    public function cancelPayoutBatch(string $payoutBatchId): bool
     {
         try {
             $params = [];
             $params["token"] = $this->_tokenCache->getTokenByFacade(Facade::Payout);
 
-            $responseJson = $this->_RESTcli->delete("payouts/".$payoutId, $params);
+            $responseJson = $this->_RESTcli->delete("payoutBatches/".$payoutBatchId, $params);
         } catch (BitPayException $e) {
-            throw new PayoutCancellationException("failed to serialize Payout object : ".$e->getMessage(), null, null, $e->getApiCode());
+            throw new PayoutBatchCancellationException("failed to serialize PayoutBatch object : ".$e->getMessage(), null, null, $e->getApiCode());
         } catch (Exception $e) {
-            throw new PayoutCancellationException("failed to serialize Payout object : ".$e->getMessage());
+            throw new PayoutBatchCancellationException("failed to serialize PayoutBatch object : ".$e->getMessage());
         }
 
         try {
             $result = json_decode($responseJson) == [];
 
         } catch (Exception $e) {
-            throw new PayoutCancellationException(
-                "failed to deserialize BitPay server response (Payout) : ".$e->getMessage());
+            throw new PayoutBatchCancellationException(
+                "failed to deserialize BitPay server response (PayoutBatch) : ".$e->getMessage());
         }
 
         return $result;
     }
 
     /**
-     * Notify BitPay Payout.
+     * Notify BitPay PayoutBatch.
      *
-     * @param  $payoutId string The id of the Payout to notify.
-     * @return array A list of BitPay Payout objects.
-     * @throws PayoutCreationException BitPayException class
+     * @param  $payoutId string The id of the PayoutBatch to notify.
+     * @return array A list of BitPay PayoutBatch objects.
+     * @throws PayoutBatchNotificationException PayoutBatchNotificationException class
      */
-    public function notifyPayout(string $payoutId): array
+    public function sendPayoutBatchNotification(string $payoutBatchId): bool
```

Comment:
this does not send a notification, it requests a notification. Please change to requestPayoutBatchNotification.
Do the same for all notification methods